Documentation/Books/Manual/Programs/Arangoinspect/Examples.md

```diff
@@ -0,0 +1,116 @@
+Arangoinspect Examples
+======================
+
+If you are asked by ArangoDB support to provide an `arangodb-inspector.json`
+file, run the _arangoinspect_ binary to generate in the current working folder.
+
+The resulting JSON file contains the names of existing databases, their
+collections, indexes, shards and some other potentially useful data about
+the system to diagnose problems.
+
+{% hint 'warning' %}
```

Comment:
please remove this block. we don't collect data. all collected is meta data. no user names no secrets, no passwords.

Reply:
As far as I've seen, some file paths are included which potentially contain the user name (which may or may not be a real name), also process information (`top`) in which user names are also listed. It's unlikely, but database, collection and index field names could contain something the user doesn't want to share. @sleto-it ?

Reply:
@Simran-B I think the _hint_ you have included about double-checking the generated info and obfuscating some of them, if needed, is a good one